### PR TITLE
Add newly shader built-ins to docs backport

### PR DESCRIPTION
**Pull request by paddy-exe**
Backport of https://github.com/godotengine/godot-docs/pull/6022
Docs of https://github.com/godotengine/godot/pull/63971

### CODE REVIEW

File: tutorials/shaders/shader_reference/spatial_shader.rst [hunk 1]
```diff
@@ -135,50 +135,58 @@ is usually:
...
++--------------------------------------+--------------------------------------------------------+
+| inout mat4 **WORLD_MATRIX**          | Model space to world space transform.                  |
++--------------------------------------+--------------------------------------------------------+
+| in mat4 **INV_CAMERA_MATRIX**        | World space to view space transform.                   |
++--------------------------------------+--------------------------------------------------------+
+| inout mat4 **PROJECTION_MATRIX**     | View space to clip space transform.                    |
++--------------------------------------+--------------------------------------------------------+
+| in mat4 **CAMERA_MATRIX**            | View space to world space transform.                   |
++--------------------------------------+--------------------------------------------------------+
+| inout mat4 **MODELVIEW_MATRIX**      |                                                        |
```
Comment (clayjohn) [inline tutorials/shaders/shader_reference/spatial_shader.rst:151]: This should keep its description

File: tutorials/shaders/shader_reference/spatial_shader.rst [hunk 2]
```diff
@@ -135,50 +135,58 @@ is usually:
...
++--------------------------------------+--------------------------------------------------------+
+| inout mat4 **INV_PROJECTION_MATRIX** | Clip space to view space transform.                    |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **NODE_POSITION_WORLD**      | Node world space position.                             |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **NODE_POSITION_VIEW**       | Node view space position.                              |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **CAMERA_POSITION_WORLD**    | Camera world space position.                           |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **CAMERA_DIRECTION_WORLD**   | Camera world space direction.                          |
```
Comment (clayjohn) [inline tutorials/shaders/shader_reference/spatial_shader.rst:161]: I know these are the same as master, but I'm just noticing that re-arranging the order would be more clear in English. I.e. ``Node world space position.`` becomes ``Node position, in world space.``

**Comment by paddy-exe** [comment]
Thanks for your feedback @clayjohn! I updated it to your feedback

**Review by clayjohn — APPROVED** [review]

**Comment by clayjohn** [comment]
Great work!

File: tutorials/shaders/shader_reference/spatial_shader.rst [hunk 3]
```diff
@@ -135,50 +135,58 @@ is usually:
...
+| inout vec3 **BINORMAL**              | Binormal in local coordinates.                         |
++--------------------------------------+--------------------------------------------------------+
+| out float **ROUGHNESS**              | Roughness for vertex lighting.                         |
++--------------------------------------+--------------------------------------------------------+
+| inout vec2 **UV**                    | UV main channel.                                       |
++--------------------------------------+--------------------------------------------------------+
+| inout vec2 **UV2**                   | UV secondary channel.                                  |
++--------------------------------------+--------------------------------------------------------+
+|| in bool **OUTPUT_IS_SRGB**          || ``true`` when calculations happen in sRGB color space |
+||                                     || (``true`` in GLES2, ``false`` in GLES3).              |
```
Comment (mhilbrunner) [inline tutorials/shaders/shader_reference/spatial_shader.rst:180]: Double `|`s

Comment (paddy-exe) [reply tutorials/shaders/shader_reference/spatial_shader.rst]: I can always count on you :D thanks for noticing

**Comment by mhilbrunner** [comment]
Thank you!

**Comment by YuriSizov** [comment]
Started the 3.6 branch with this PR merged, and reverted it in the 3.5 branch. Let me know if I everything is good. 🙂

**Comment by paddy-exe** [comment]
@YuriSizov Seems good to me! Sorry, should have specified the PR for 3.6

**Pull request merged**